### PR TITLE
Add drops/actions/convert-route endpoint

### LINKED ISSUE #283: [API parity] Document /drops/actions/convert-route
Landed in the API via https://github.com/azurestandard/beehive/pull/4738.

This is for moving a drop from one route to another.

It's being used [here in the Logistics app](https://github.com/azurestandard/internal-website/blob/master/src/modules/logistics/controllers/logistics.js#L205-L216).

### PR DESCRIPTION
**Pull request by igregson**
Resolves #283

### CODE REVIEW

**Comment by igregson** [comment]
Wasn't sure who to request a review from for this. Went with you, David, since you've generally reviewed my previous API-Spec PRs. Since you worked on this code in the backend, Tom, thought maybe that your review on it would also/instead be appropriate.

File: spec.json [hunk 1]
```diff
@@ -8964,6 +8964,44 @@
...
+            "name": "targetRouteId",
+            "description": "The ID of the route that the drop(s) should be moved to."
+          },
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "dropIds",
+            "description": "A comma-separated list of the IDs of drops to be moved."
```
Comment (davidmcatee-azure) [inline spec.json:9000]: According to the code, `dropIds` is not a comma-separated list of ids, but rather an array of ids.

**Review by davidmcatee-azure — CHANGES_REQUESTED** [review]

File: spec.json [hunk 2]
```diff
@@ -8964,6 +8964,47 @@
...
+          "200": {
+            "description": "OK"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "type": "integer"
+            },
+            "in": "query",
```
Comment (davidmcatee-azure) [inline spec.json:8982]: Just noticed that these parameters should be in the request body, not the query parameters.

**Review by davidmcatee-azure — CHANGES_REQUESTED** [review]

File: spec.json (hunk 2 above)
Comment (igregson) [reply spec.json]: 🤦 

Good catch. Fixed.

**Review by davidmcatee-azure — APPROVED** [review]

**Pull request merged**